mslib/utils.py

```diff
@@ -504,6 +504,9 @@ def path_points(points, numpoints=100, connection='linear'):
     """
     if connection not in ['linear', 'greatcircle']:
         return None, None
+    if points is None or len(points) == 0:
```

Comment:
two None or three None??

Reply:
this what creates this state may need a refactoring. I found points = None and sometimes points = []

Reply:
I mean, line 506 is `return None, None`. Line 508 is `return None, None, None`. I have doubts that both are correct. Maybe neither...?

Reply:
I should have added 
# ToDo refactor soon there.
I was surprised about the two kind of states.